docs/source/install.rst

```diff
@@ -6,52 +6,63 @@ Install
 Currently installation is supported by constructing a conda_ environment. You need to be running
 a conda_ package (eg. miniconda, anaconda, etc) to install autometa.
 
-Conda environment creation:
-
-.. code-block:: bash
-
-    conda create -n autometa "python>=3.7"
-    conda install -n autometa -c bioconda -c conda-forge \
-        biopython \
-        pandas \
-        tqdm \
-        tsne \
-        numpy \
-        scikit-learn \
-        scikit-bio \
-        samtools \
-        bedtools \
-        bowtie2 \
-        hmmer \
-        prodigal \
-        diamond \
-        ndcctools \
-        parallel \
-        requests \
-        umap-learn \
-        hdbscan
+Conda installation
+==================
 
+#. Install miniconda_
+#. Create a new environment ``conda create -n autometa "python>=3.7"``
+#. Install autometa ``conda install -c conda-forge -c bioconda autometa --yes``
+#. Actiavate autometa envoirnonment ``conda activate autometa``
```

Comment:
NOTE: We have requirements.txt in our root directory. So the environment can be created and installed with one command:

## Install from source (using make)

```bash
cd $HOME
git clone https://github.com/KwanLab/Autometa.git
cd $HOME/Autometa
# create autometa conda environment
make create_environment
# activate env
conda activate autometa
# install autometa source code
make install
```

## Install from source (full commands)

```bash
cd $HOME
git clone https://github.com/KwanLab/Autometa.git
cd $HOME/Autometa
# Construct the environment from the listed requirements.
conda create -n autometa --file=requirements.txt
# Install the autometa code base from source
python setup.py install
```

Reply:
## build the docker image from source

Also note, you can building a docker image for your clone of the Autometa repo.

```bash
cd $HOME
git clone https://github.com/KwanLab/Autometa.git
cd $HOME/Autometa
# This will tag the image as jason-c-kwan/autometa:<your current branch>
make image
```